### PR TITLE
SDKCF-3707: Integration tests fixes

### PR DESCRIPTION
**Pull request by malayli**
# Description
Integration tests fixes.

## Links
SDKCF-3707

# Checklist
- [x] I have read the [contributing guidelines](../CONTRIBUTING.md).
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data **before every commit**, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

File: IntegrationTests/IntegrationTests.swift [hunk 1]
```diff
@@ -42,10 +42,10 @@ class IntegrationTests: XCTestCase {
                 response = nil
             }
             XCTAssertNotNil(response)
-            XCTAssert(response?.enabled == true)
-            XCTAssertNotNil(response?.endpoints.ping)
-            XCTAssertNotNil(response?.endpoints.displayPermission)
-            XCTAssertNotNil(response?.endpoints.impression)
+            XCTAssert(response?.rolloutPercentage == 100)
```
Comment (imaginaris) [inline IntegrationTests/IntegrationTests.swift]: Can we assume that rollout will always be 100? Maybe just check if it's not nil or >0?

Comment (malayli) [reply IntegrationTests/IntegrationTests.swift]: Checking not nil is better I think.

Comment (imaginaris) [reply IntegrationTests/IntegrationTests.swift]: Ah right it can be 0. I think checking for nil is enough

**Review by imaginaris — APPROVED** [review]

**Comment by donnie-jp** [comment]
Thanks for fixing these @malayli

**Pull request merged**